.pre-commit-config.yaml

```diff
@@ -13,11 +13,10 @@ repos:
       - id: pretty-format-json
         exclude: ^$|.devcontainer
 
-  - repo: https://github.com/asottile/setup-cfg-fmt
-    rev: v2.2.0
+  - repo: https://github.com/abravalheri/validate-pyproject
+    rev: v0.12.1
```

Comment:
I will review later, but it would be nice if we had a autoformatter for the `pyproject.toml`

Reply:
I find that its edits are a bit heavy-handed so I tend not to use it.
* https://github.com/kornia/kornia/pull/2225/commits/50ad3c3a9ea332ac5fd30d4715a1c49f6da9a901